samples/dnn/squeezenet_halide.cpp

```diff
@@ -79,8 +79,7 @@ int main(int argc, char **argv)
         exit(-1);
     }
 
-    resize(img, img, Size(227, 227));                // SqueezeNet v1.1 predict class by 3x227x227 input image.
-    Mat inputBlob = blobFromImage(img, 1.0, Size(), Scalar(), false);  // Convert Mat to 4-dimensional batch.
+    Mat inputBlob = blobFromImage(img, 1.0, Size(227, 227), Scalar(), false);  // Convert Mat to 4-dimensional batch.
```

Comment:
Please consider the last flag `crop` of `blobFromImage`. By default it's true that means we keep aspect ratios of image by cropping some pixels from the highest dimension. So to get similar input we should pass `false` value. https://docs.opencv.org/master/d6/d0f/group__dnn.html#ga33d1b39b53a891e98a654fdeabba22eb